google/cloud/forseti/services/inventory/storage.py

```diff
@@ -331,7 +333,9 @@ def from_resource(cls, index, resource):
             resource (Resource): Crawled resource.
 
         Returns:
-            object: database row object.
+            Tuple[list, list]: A tuple containing a single row for the main
```

Comment:
resource row is of type dict instead of list

Reply:
Fixed.